### DISCUSSION TITLE
Confirm closing tab when currently writing or editing issue or comment text

### DISCUSSION THREAD

**Discussion by 0liver** [community/community#14548] [Projects and Issues]
I've just lost a bunch of my work that I've put into a comment I was editing, because my finger slipped on the `W` key while I was trying to mark a piece of text as code (with `Ctrl` + `E`).

I was very surprised that the tab simply closed without any confirmation message.

Is this a bug or has this feature never been available?

It seems quite necessary to me - while writing up this thread, it happened to me again!

**Comment by dsyrstad** [comment]
This also happens on Projects if you're editing a card/issue description and click on a different card, the new card is displayed and your edits are lost. Unless you hit the "Update comment" button, they are lost.

This is different than editing the same issue in Issues. When you edit in Issues and don't click "Update comment", go to a different issue, then come back to the one you were editing and choose Edit from the menu, you will see your unsaved edits. This behavior doesn't happen in Projects and I've lost a number of updates due to looking at a related card, etc.
